sdk/src/claim.rs

```diff
@@ -236,9 +236,10 @@ pub enum AssertionStoreJsonFormat {
 /// Remote manifest options. Use 'set_remote_manifest' to generate external manifests.
 #[derive(Clone, Debug, PartialEq)]
 pub enum RemoteManifest {
-    NoRemote,       // No external manifest (default)
+    NoRemote,                // No external manifest (default)
```

Comment:
any reason for the extra space?

Reply:
every time I remove it cargo fmt puts it back